.github/workflows/testflight.yml

```diff
@@ -0,0 +1,106 @@
+name: Distribute TestFlight Build
+
+# on:
+#  workflow_dispatch: 
+#
+# on:
```

Comment:
> Discuss and agree on what should be triggers to run this GA (merge to /release/* branch, cron with daily / weekly builds, merging to master, "dispatch_workflow" with some user-based limitations)

I think that depends on who is going to use those builds. If it's primarily maintainers, then I think it's ok to build it on every `master` commit.

Reply:
Posted new Notion page with voting on that 🙏🏻